worker-languagedetection/supported-languages-fasttext.md

```diff
@@ -0,0 +1,180 @@
+## Supported Languages
```

Comment:
Should this file not just be called "supported-languages.md" as we are not doing multiple implementations?

Reply:
Done